modules/modal/css/README.md

```diff
@@ -0,0 +1,37 @@
+# Canvas Kit Modal
+
+A Modal component that allows you to render a Popup with a translucent overlay.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-modal
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-modal/index.scss';
+```
+
+## Usage
+
+Use `.wdc-modal` to create a modal container with the translucent overlay. Use `.wdc-popup` within
+the `.wdc-modal` to create the popup. See [Popup](?path=/story/css-popup--all) for instructions on
```

Comment:
What's the proper way to link to another component in the README? I noticed a few of the READMEs use the following syntax...

`[canvas-kit-core](../../core/css)`

... but the links don't actually work (at least not locally).